packages/training/python/scripts/hf/trajectories_to_hf_dataset.py

```
@@ -0,0 +1,781 @@
+    # Also save as parquet for easy inspection
+    for split_name, split_data in full_dataset.items():
+        parquet_path = output_path / f"{split_name}.parquet"
+        split_data.to_parquet(str(parquet_path))
+        logger.info(f"Saved {split_name}.parquet")
+    
+    # Push to hub if requested
+    if config.push_to_hub:
+        push_to_huggingface(full_dataset, config)
+
+
+def push_to_huggingface(dataset: "DatasetDict", config: ExportConfig):
```

Ruff (0.14.13)

Undefined name `DatasetDict`

(F821)